__tests__/database.test.js

```diff
@@ -69,7 +69,8 @@ describe("GET articles", () => {
             body: expect.any(String),
             created_at: expect.any(String),
             votes: expect.any(Number),
-            article_img_url: expect.any(String)
+            article_img_url: expect.any(String),
+            comment_count: expect.any(Number)
```

Comment:
this could also be its own test, but its fine going back and changing this one if you prefer